EXPLAINER.md

```diff
@@ -258,6 +258,34 @@ From speaking about the feature at multiple events over the years, it is clear
 that web developers have been quite positive about the ability to use NFC from
 the web.
 
+## Changes done after Origin Trials (OT)
```

Comment:
```suggestion
## Changes done after Origin Trial (OT)

Reply:
On the Chrome page its in plural, that's why I used it (confused).